manim/mobject/geometry.py

```diff
@@ -1269,10 +1328,16 @@ def get_default_tip_length(self) -> float:
     def set_stroke_width_from_length(self):
         """Used internally. Sets stroke width based on length."""
         max_ratio = self.max_stroke_width_to_length_ratio
-        self.set_stroke(
-            width=min(self.initial_stroke_width, max_ratio * self.get_length()),
-            family=False,
-        )
+        if config.renderer == "opengl":
+            self.set_stroke(
+                width=min(self.initial_stroke_width, max_ratio * self.get_length()),
+                recurse=False,
+            )
+        else:
+            self.set_stroke(
+                width=min(self.initial_stroke_width, max_ratio * self.get_length()),
+                family=False,
+            )
```

Comment:
Nonblocking, but it might be beneficial to just make the non-opengl implementation of this use `recurse` instead

Reply:
I was thinking about that and even started to change the keyword name -- but then I had to change this in a *lot* of places for a lot of methods to keep it somewhat consistent.

I think it is worth it, overall, but it should be done cleanly; possibly in a separate PR with a proper keyword argument deprecation?

Reply:
I'm not sure many end users are touching the `family` argument, so I don't think it would require deprecation and could just be changed wholesale, but wanting it in a separate PR makes sense